common/ts/prepare-task.ts

```diff
@@ -30,6 +30,7 @@ export default async function prepareTask(endpoint: Endpoint, rootPath: string)
 
   if (await branchFeatureSupported(endpoint)) {
     await populateBranchAndPrProps(props);
+    props['sonar.scanner.metadataFilePath'] = reportPath();
```

Comment:
I find it misleading. At least put a comment saying that branchFeatureSupported is by chance checking that version is > 7.2.

Reply:
Ok that has been added. Hardening soon to fix this.